src/custom-lists/background/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ListObject {
```

Comment:
I would drop the `Object` at the end of these names; it's implied that these are object types. Maybe `PageList` and `PageListEntry` would be good names? (We already have a `Page` type which is arguably our core data type in our data model)